src/analysis.rs

```diff
@@ -217,29 +254,31 @@ where
             path.retain(|c| c != ':');
             path.push_str(&entry.location.file);
 
+            // can't do this in one statement because it deadlocks
+            let v = cache.lock().get(&entry.location.file);
```

Comment:
Could we use the entry `audio_id` instead? It looks like a hash of the audio content, calculated by Traktor. I think it would be more resilient on file moves